spec/presenters/hyku/file_set_presenter_spec.rb

```diff
@@ -5,10 +5,17 @@
   let(:request) { double(base_url: 'http://test.host') }
   let(:presenter) { described_class.new(solr_document, nil, request) }
   let(:id) { CGI.escape(file_set.original_file.id) }
+
+  let(:file_set_not_image) { FactoryGirl.create(:file_set) }
+  let(:solr_document_not_image) { SolrDocument.new(file_set_not_image.to_solr) }
+  let(:presenter_not_image) { described_class.new(solr_document_not_image, nil, request) }
```

Comment:
Can this just be the presenter in a context that is about "a file that is not an image"? For example:
```ruby
describe "#display_image" do
  subject { presenter.display_image }
  context "when the file is not an image" do
    before do
       Hydra::Works::AddFileToFileSet.call(file_set,
          fixture_file('csv/sample.csv'), :original_file)
     end
     it { is_expected.to be nil }
  end

  context "when the file is an image" do
    before do
       Hydra::Works::AddFileToFileSet.call(file_set,
          fixture_file('images/world.png'), :original_file)
     end
    ...
  end
end
```

this is preferable in two regards:
1. it's faster because we don't have to set up two sets of objects for each test.
1. we don't need to keep track of a second definition of presenter and solr_document.